styles/components/HackathonModal.module.scss

```diff
@@ -0,0 +1,81 @@
+// Import fonts
+@import url('https://fonts.cdnfonts.com/css/motorblock');
+@import url('https://fonts.googleapis.com/css2?family=Orbitron:wght@400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+.Modal {
+  @apply w-11/12 max-w-2xl outline-none overflow-hidden;
+  @apply bg-opacity-0;
+}
+
+.modalImageContainer {
+  @apply absolute w-52 h-72 z-40;
+  @apply top-8 left-1/2;
+  @apply animate-float;
+}
+
+.modalContent {
+  @apply fixed top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-[40%] rounded-2xl shadow-md p-8 w-11/12 max-w-2xl z-50 aspect-[1.5] outline-none;
```

Comment:
can you break this up into smaller `@apply` statements with like wise properties.